src/cigar.py

```diff
@@ -33,7 +34,14 @@ def cigar_to_edits(cigar: str) -> str:
     'MDMMMMMMIMMMM'
 
     """
-    return ""
+
+    edits = ''
+    pairs = split_pairs(cigar)
+
+    for tuple in pairs:
```

Comment:
I would highly recommend that you do not use `tuple` as a variable name. You will over-shadow the builtin type, and that will one-day cause problems.